Dockerfile

```diff
@@ -31,6 +31,13 @@ ADD setup.sh /setup.sh
 RUN chmod +x /setup.sh
 RUN /setup.sh
 
+ADD locale.gen /etc/locale.gen
+RUN /usr/sbin/locale-gen
+ENV LANG en_US.UTF-8
+ENV LANGUAGE en_US:en
+ENV LC_ALL en_US.UTF-8
+RUN update-locale ${LANG}
+
```

Comment:
aren't these all defaults? why do we need to add them explicitly? To give users a place to change them if they want?

Reply:
They are added here to generate locale for the whole system and on the DB side they are then specified here https://github.com/kartoza/docker-postgis/blob/develop/setup-database.sh#L26
with the env variables controlled here https://github.com/kartoza/docker-postgis/blob/develop/env-data.sh#L101